apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -197,7 +224,48 @@ NetSimRouterLogModal.prototype.isVisible = function () {
  * Fill the root div with new elements reflecting the current state
  */
 NetSimRouterLogModal.prototype.render = function () {
-  // Be lazy, don't render if not visible.
+  if (usingNewLogBrowser()) {
+    this.newRender_();
+  } else {
+    this.oldRender_();
+  }
+};
+
+NetSimRouterLogModal.prototype.newRender_ = function () {
```

Comment:
how confident are we that we will soon be able to remove the old functionality? this naming scheme seems fine enough as a temporary placeholder, but I'd hate to see it linger